backend/main.py

```
@@ -32,38 +30,74 @@ class DevRAIApplication:
+            # Discord Mode
+            if settings.discord_enabled:
+                # 1. Start queue 
+                if settings.code_intelligence_enabled:
+                    await self.queue_manager.start(num_workers=3)
+
+                # 2. Discord
+                try:
+                        await self.discord_bot.load_extension("integrations.discord.cogs")
+                        asyncio.create_task(
+                            self.discord_bot.start(settings.discord_bot_token)
+                        )
```

Ruff (0.14.14)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)